contracts/services/splitter/src/tests.rs

```diff
@@ -245,6 +246,46 @@ fn instantiate_fails_for_invalid_split_config() {
     suite.splitter_init(&cfg);
 }
 
+#[test]
+#[should_panic(expected = "Configuration error: Invalid split config: amount cannot be 0.")]
+fn instantiate_fails_for_zero_amount() {
+    let mut suite = SplitterTestSuite::default();
+
+    let input_addr = suite.account_init("input_account", vec![]);
+
+    // Configure splitter with invalid split config
+    let split_cfg = UncheckedSplitConfig::new(
+        UncheckedDenom::Native(NTRN.into()),
+        &input_addr,
+        Some(Uint128::zero()),
+        None,
+    );
+    let cfg = suite.splitter_config(vec![split_cfg]);
+
+    // Instantiate Splitter contract
+    suite.splitter_init(&cfg);
+}
+
+#[test]
+#[should_panic(expected = "Configuration error: Invalid split config: ratio cannot be 0.")]
+fn instantiate_fails_for_zero_ratio() {
+    let mut suite = SplitterTestSuite::default();
+
+    let input_addr = suite.account_init("input_account", vec![]);
+
+    // Configure reverse splitter with invalid split config
+    let split_cfg = UncheckedSplitConfig::new(
+        UncheckedDenom::Native(NTRN.into()),
+        &input_addr,
+        None,
+        Some(UncheckedRatioConfig::FixedRatio(Decimal::zero())),
+    );
+    let cfg = suite.splitter_config(vec![split_cfg]);
+
+    // Instantiate Splitter contract
+    suite.splitter_init(&cfg);
+}
+
 #[test]
 #[should_panic(
     expected = "Configuration error: Duplicate split 'Native(\"untrn\")|AccountAddr(\"cosmwasm1ea6n0jqm0hj663khx7a5xklsmjgrazjp9vjeewejn84sanr0wgxq2p70xl\")' in split config."
```

Comment:
I changed this line because I renamed an enum variant from `AccountAddr` to `Addr`, 
Would be great if we can avoid those `should_panic` when we need to include some parsing of enums or alike.